### PR TITLE
Add decimal digit token filter docs #7923

### PR DESCRIPTION
**Pull request by AntonEliatra**
### Description
Add decimal digit token filter docs

### Issues Resolved
Partly closes https://github.com/opensearch-project/documentation-website/issues/7923

### Version
all

### Checklist
- [x] By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license and subject to the [Developers Certificate of Origin](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

**Comment by AntonEliatra** [comment]
I added -- around the number range (0-9) to fix vale errors, but don't know if it looks ok now

File: _analyzers/token-filters/decimal-digit.md [hunk 1]
```diff
@@ -0,0 +1,88 @@
+---
+layout: default
+title: Decimal digit
+parent: Token filters
+nav_order: 80
+---
+
+# Decimal digit token filter
+
+The `decimal_digit` token filter in OpenSearch is used to normalize characters that are decimal digits (--0-9--) from various scripts into their ASCII equivalents. This is useful when you want to ensure that all digits are treated uniformly in text analysis, regardless of the script in which they are written.
```
Comment (kolchfa-aws) [inline _analyzers/token-filters/decimal-digit.md:10]:
```suggestion
The `decimal_digit` token filter in OpenSearch is used to normalize characters that are decimal digits (0--9) from various scripts into their ASCII equivalents. This is useful when you want to ensure that all digits are treated uniformly in text analysis, regardless of the script in which they are written.
```

**Comment by kolchfa-aws** [comment]
@AntonEliatra The range should be `0--9`. I updated it

**Comment by kolchfa-aws** [comment]
@udabhas Could you review this PR for technical accuracy?

**Comment by vagimeli** [comment]
@varun-lodaya The documentation is awaiting tech review and approval, which is delaying progress. Could you please suggest alternative reviewers who can assist with this task in a timely manner? We're eager to move this forward. Thank you.

**Comment by vagimeli** [comment]
> @varun-lodaya The documentation is awaiting tech review and approval, which is delaying progress. Could you please suggest alternative reviewers who can assist with this task in a timely manner? We're eager to move this forward. Thank you.

@varun-lodaya This is over a month old. We need tech review approval to move it forward in the documentation process. Please review this week or provide a peer who can review it. Thank you.

**Review by udabhas — APPROVED** [review]
Looks good to me!

File: _analyzers/token-filters/decimal-digit.md [hunk 2]
```diff
@@ -0,0 +1,88 @@
+---
+layout: default
+title: Decimal digit
+parent: Token filters
+nav_order: 80
+---
+
+# Decimal digit token filter
+
+The `decimal_digit` token filter in OpenSearch is used to normalize characters that are decimal digits (0--9) from various scripts into their ASCII equivalents. This is useful when you want to ensure that all digits are treated uniformly in text analysis, regardless of the script in which they are written.
```
Comment (kolchfa-aws) [inline _analyzers/token-filters/decimal-digit.md:10]:
```suggestion
The `decimal_digit` token filter is used to normalize characters that are decimal digits (0--9) into their ASCII equivalents in various scripts. This is useful when you want to ensure that all digits are treated uniformly in text analysis, regardless of the script in which they are written.
```

File: _analyzers/token-filters/decimal-digit.md [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+
+
+## Example
+
+The following example request creates a new index named `my_index` and configures an analyzer with `decimal_digit` filter:
```
Comment (kolchfa-aws) [inline _analyzers/token-filters/decimal-digit.md:15]:
```suggestion
The following example request creates a new index named `my_index` and configures an analyzer with a `decimal_digit` filter:
```

**Review by kolchfa-aws — APPROVED** [review]
Thank you, @AntonEliatra! A couple of small changes.

File: _analyzers/token-filters/decimal-digit.md [hunk 4]
```diff
@@ -0,0 +1,88 @@
+---
+layout: default
+title: Decimal digit
+parent: Token filters
+nav_order: 80
+---
+
+# Decimal digit token filter
+
+The `decimal_digit` token filter is used to normalize characters that are decimal digits (0--9) into their ASCII equivalents in various scripts. This is useful when you want to ensure that all digits are treated uniformly in text analysis, regardless of the script in which they are written.
```
Comment (natebower) [inline _analyzers/token-filters/decimal-digit.md:10]:
```suggestion
The `decimal_digit` token filter is used to normalize decimal digit characters (0--9) into their ASCII equivalents in various scripts. This is useful when you want to ensure that all digits are treated uniformly in text analysis, regardless of the script in which they are written.
```

**Review by natebower — COMMENTED** [review]
@AntonEliatra @kolchfa-aws Just one minor change. Thanks!

**Pull request merged**